### ISSUE TITLE
Not calling automat/hmdb on related_to

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/strider#63]
Here's the query:

```
'message': {'message': {'query_graph': {'nodes': {'a': {'category': 'biolink:ChemicalSubstance',
      'id': 'CHEBI:88916'},
     'b': {'category': 'biolink:Disease'}},
    'edges': {'ab': {'subject': 'a',
      'object': 'b',
      'predicate': 'biolink:related_to'}}}}}
```

biolink:related_to is in automat/hmdb's predictes:
```
{
  "biolink:ChemicalSubstance": {
    "biolink:Gene": [
      "biolink:interacts_with"
    ],
    "biolink:GeneOrGeneProduct": [
      "biolink:interacts_with"
    ],
    "biolink:MacromolecularMachine": [
      "biolink:interacts_with"
    ],
    "biolink:Disease": [
      "biolink:related_to"
    ],
...
```

There is an edge in hmdb that links this input chemical to MONDO:0011122 (obesity disorder) that is not being returned.  Looking at the logs from strider, it appears that automat/hmdb is not being contacted.

**Comment by cbizon** [comment]
Also not calling hmdb for biolink:interacts_with for ChemicalSubstance to Gene:

```
'message': {'message': {'query_graph': {'nodes': {'a': {'category': 'biolink:ChemicalSubstance',
      'id': 'CHEBI:34253'},
     'b': {'category': 'biolink:Gene'}},
    'edges': {'ab': {'subject': 'a',
      'object': 'b',
      'predicate': 'biolink:interacts_with'}}}}}
```

**Comment by patrickkwang** [comment]
The first example here was fixed by #81.

**Comment by patrickkwang** [comment]
All fixed by #83.
